docs/content/opa-1.md

```diff
@@ -0,0 +1,201 @@
+
+1. A new `rego.v1` import has been introduced that, when used, makes OPA apply all restrictions that will eventually be enforced by default in OPA v1.0.
+   If a Rego module imports `rego.v1`, it means the `if` and `contains` keywords are required when declaring rules. Constants, rules that only consist of a value assignment, are exempted.
+2. The `--rego-v1` flag on the `opa fmt` command will rewrite existing modules to use the `if` and `contains` keywords where applicable.
+3. The `--rego-v1` flag on the `opa check` command will check that the `if` and `contains` keywords are used where applicable in a module.
+
+### Backwards compatibility in OPA v1.0
+
+By default, OPA v1.0 will require the usage of `if` and `contains` as described above, but to maintain backward compatibility for policies written pre-OPA v1.0, a new flag called `--v0-compatible` will be added to OPA commands such as `opa run`, to avoid breaking existing policies and give users time to update them. 
+Similar functionality will be added to OPA’s Go SDK and Go API to assist users that embed OPA as a library in their Go services and also to OPA’s build command.
+
+## Prohibit duplicate imports
+
+The Rego compiler supports [strict mode](../policy-language/#strict-mode) which enforces additional constraints and safety checks during compilation. 
+One of these checks is to prohibit duplicate imports where one import shadows another. OPA v1.0 will enforce this check by default.
+
+An import shadowing another is most likely an authoring error and probably unintentional. OPA checking this by default will help to avoid policy evaluations resulting in error-prone decisions.
+
+### Making new and existing policies compatible with OPA v1.0
+
+1. A new `rego.v1` import has been introduced that, when used, makes OPA apply all restrictions that will eventually be enforced by default in OPA v1.0.
+   If a Rego module imports `rego.v1`, duplicate imports are prohibited.
+2. The `--rego-v1` flag on the `opa fmt` command will reject modules with duplicate imports.
+3. The `--rego-v1` flag on the `opa check` command will check that duplicate imports are not present in a module.
+
+### Backwards compatibility in OPA v1.0
+
+If pre-OPA v1.0 behavior is desired where this check is only enforced when `strict mode` is enabled, a new `--v0-compatible` flag will be added to the OPA CLI to achieve that.
+Similar functionality will be added to OPA’s Go SDK, Go API and build command.
+
+## `input` and `data` keywords are reserved
+
+The Rego compiler supports [strict mode](../policy-language/#strict-mode), which enforces additional constraints and safety checks during compilation. 
+One of these checks is to ensure that `input` and `data` are reserved keywords and may not be used as names for rules and variable assignments.
+
+The `input` document holds the user-provided input, while the data pushed into OPA and rule evaluation results are nested under the `data` document.
```

Comment:
In some of the repos where I've helped teams enforce strict mode, they have not understood that `with input as { .. }` doesn't constitute shadowing. Perhaps it would be good to say something to that end here.